lib/srv/desktop/tdp/proto.go

```diff
@@ -232,6 +250,68 @@ func decodePNGFrame(in peekReader) (PNGFrame, error) {
 	return PNGFrame{Img: img}, nil
 }
 
+// PNG2Frame is a newer versin of PNGFrame that includes the
+// length of the PNG data.
+type PNG2Frame PNGFrame
```

Comment:
Is there a particular reason to keep the `PNGFrame` around now that we're no longer using it? Seems like we can just rename it to `PNG2Frame`, and delete all the `PNGFrame` related code.

Reply:
It may still exist in old session recordings.

Reply:
Oh, right.